backend/bom-bom-server/src/main/java/me/bombom/api/v1/common/config/SecurityConfig.java

```diff
@@ -0,0 +1,39 @@
+package me.bombom.api.v1.common.config;
+
+import lombok.RequiredArgsConstructor;
+import me.bombom.api.v1.auth.handler.OAuth2LoginSuccessHandler;
+import me.bombom.api.v1.auth.service.CustomOAuth2UserService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.config.annotation.web.configurers.HeadersConfigurer.FrameOptionsConfig;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+@RequiredArgsConstructor
+public class SecurityConfig {
+
+    private final CustomOAuth2UserService customOAuth2UserService;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                // 모든 설정 열어둠
+                .csrf(AbstractHttpConfigurer::disable)
+                .headers(headers -> headers.frameOptions(FrameOptionsConfig::disable))
```

Comment:
p1

프론트엔드가 로컬에서도 실제 API호출해서 테스트 해볼 수 있도록 CORS 설정도 해주면 좋을 것 같습니다.  

예를들어 백엔드 서버가 api.bombom.news에 있고 프론트엔드가 localhost:3000에서 동작할 때 이 둘은 서로 다른 출처로 간주되기 때문에, 별도의 CORS 설정이 없으면  브라우저가 AJAX 요청을 차단해 버립니다.

아래와 같이 추가해주면 좋을 것 같아요


```suggestion
                .csrf(AbstractHttpConfigurer::disable)
                .cors(configurer ->  configurer.configurationSource(corsConfigurationSource()))
                .headers(headers -> headers.frameOptions(FrameOptionsConfig::disable))      
```

```java
@Bean
  public CorsConfigurationSource corsConfigurationSource() {
    CorsConfiguration configuration = new CorsConfiguration();
    configuration.addAllowedOriginPattern("*"); // 모든 origin 허용, 필요에 따라 특정 origin으로 변경 가능
    configuration.addAllowedMethod("*");
    configuration.addAllowedHeader("*"); // 모든 헤더 허용
    configuration.setAllowCredentials(true);
    configuration.setMaxAge(3600L);

    UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
    source.registerCorsConfiguration("/**", configuration);
    return source;
  }
```